tests/unit/components/sl-tooltip-test.js

```diff
@@ -52,6 +52,24 @@ test( '"title" property needs to be a string', function( assert ) {
         'Property was empty'
     );
 
+    // Symbol Property
+
+    assertionThrown = false;
+
+    try {
+        let sym = Symbol("foo");
```

Comment:
Modify to follow these style guidelines:
* https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#block-statements
* https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#strings-1